server/secrets.py

```diff
@@ -1,7 +1,10 @@
 MBTA_V3_API_KEY = ''
 POSTGRES_DB = 'newtrains'
-POSTGRES_USER = 'ubuntu'  # Update to your Postgres user account if not using ubuntu
-POSTGRES_PASS = ''  # Add password if required; Uses account auth on linux
+# POSTGRES_HOST = 'postgres'
+
+# current set to root as the postgres db on docker doesn't by default create a separate user
+POSTGRES_USER = 'root'  # Update to your Postgres user account if not using ubuntu
+POSTGRES_PASS = 'password'  # Add password if required; Uses account auth on linux
```

Comment:
are these secrets used for other purposes? if so I can add the docker version to the env set as well.

Reply:
@mathcolo